Packs/Carbon_Black_Enterprise_Response/ReleaseNotes/2_1_18.md

```diff
@@ -0,0 +1,5 @@
+
+#### Integrations
+##### VMware Carbon Black EDR v2
+- Converted IP Addresses from integers to dotted quad strings, (e.g 8.8.8.8)
```

Comment:
Please look at [here](https://xsoar.pan.dev/docs/documentation/release-notes#general) and change the release note accordingly